lib/msal-common/src/server/ServerCodeRequestParameters.ts

```diff
@@ -63,9 +66,9 @@ export class ServerCodeRequestParameters extends ServerRequestParameters {
      * Check to see if there are SSO params set in the Request
      * @param request
      */
-    isSSOParam(account: Account) {
+    isSSOParam() {
```

Comment:
I feel like maybe this should be named `hasSsoParam` instead of `isSSOParam`? Doesn't make grammatically to me.

Reply:
Changed in a future PR.